cmd/jaeger/internal/extension/jaegerquery/factory.go

```diff
@@ -27,13 +28,18 @@ func NewFactory() extension.Factory {
 
 func createDefaultConfig() component.Config {
 	return &Config{
-		HTTP: confighttp.ServerConfig{
-			Endpoint: ports.PortToHostPort(ports.QueryHTTP),
+		QueryOptionsBase: app.QueryOptionsBase{
+			BasePath: "/",
```

Comment:
this looks new

Reply:
@yurishkuro I added this in to match the default of v1 from https://github.com/jaegertracing/jaeger/blob/main/cmd/query/app/flags.go#L99. But it actually looks like that's being done in https://github.com/jaegertracing/jaeger/blob/main/cmd/query/app/static_handler.go#L126-L128 anyway. I'll remove it.